docs/api-docs/catalog/price-list-overview.md

```diff
@@ -141,20 +142,20 @@ See [Channels, Sites and Routes](https://developer.bigcommerce.com/api-reference
 ```
 
 
-## Price List Notes
+## Price List notes
 
 - Bulk Pricing Tiers may additionally be associated with a Price Record to indicate different pricing as the quantity in cart increases.
 
-- If a variant has a `Price Record` any existing product-level bulk pricing will not apply in the cart.  For variants without `Price Records`, any existing product bulk pricing will apply.
+- If a variant has a `Price Record`, any existing product-level bulk pricing will not apply in the cart.  For variants without `Price Records`, any existing product bulk pricing will apply.
 
-- Price Lists Records accepts bulk upsert. Only one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) can done at a time. Running more than one in parallel on the **same store** will cause a 429 error and the request will fail.
+- Price Lists Records accepts bulk upsert. You can only do one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) at a time. Running more than one in parallel on the **same store** will cause a 429 error, and the request will fail.
 
-## Price List Resources
+## Price List resources
 
-### Related Endpoints
+### Related endpoints
 * [Get Price List Collection](/api-reference/catalog/pricelists-api/price-lists/getpricelistcollection)
 
-### Webhooks Available
+### Webhooks available
 
 There are no direct webhooks available for Price Lists. Since Price Lists directly relate to products, webhooks related to products will fire for corresponding changes such as pricing.

```

Comment:
A price record update will not trigger product or variant webhooks.  Webhooks for price lists and price records are on our roadmap, but no idea when that will be done.